evap/results/views.py

```diff
@@ -67,15 +67,12 @@ def course_detail(request, semester_id, course_id):
     else:
         public_view = request.GET.get('public_view') == 'true'  # if parameter is not given, show own view.
 
-    # If grades are not published, there is no public view
-    if not course.has_enough_voters_to_publish_grades:
+    if not course.can_publish_rating_results:
```

Comment:
could you please add a comment to explain why this is necessary? (It's about the state of the toggle-button which should not say public if there is no public view of this page)